crates/conjure_core/src/parse/parse_model.rs

```diff
@@ -344,7 +350,35 @@ fn parse_domain_value_int(obj: &JsonValue) -> Option<i32> {
         Some(-inner_num)
     }
 
-    try_parse_positive_int(obj).or_else(|| try_parse_negative_int(obj))
+    // it will be too annoying to store references inside all our int domain ranges, so just
+    // resolve them here.
+    //
+    // this matches savilerow, where lettings must be declared before they are used.
```

Comment:
letting must indeed be defined before they are referenced, but we shouldn't necessarily have to inline them in the long run. can you please edit the comment here ot say we should add references to domains in the future? thanks.

Reply:
done, thanks